src/test/resources/velocity/velocity-test-engine-fail.properties

```diff
@@ -0,0 +1,3 @@
+runtime.references.strict: true
+file.resource.loader.path: target/test/VelocityTemplateResolverSpec/template/
+template.file.name: LicenseFailTest.html
```

Comment:
To have the specs for a test in one place, rather inline the content of such small files and write them into a temporary file under something like `target/test/testid`, clean them up at the end of the test.